src/main/webapp/js/instructorFeedbackEdit.js

```diff
@@ -1415,7 +1415,15 @@ function setupFsCopyModal() {
                 for (var i = 0 ; i < coursesTable.length; i++) {
                     htmlToAppend += "<div class=\"checkbox\">";
                     htmlToAppend += "<label><input type=\"checkbox\" name=\"copiedcoursesid\"";
-                    htmlToAppend += "value=\"" + coursesTable[i].id + "\"> [" + coursesTable[i].id + "] : " + coursesTable[i].name;
+                    if (String(coursesTable[i].id) == courseid){
+                    	htmlToAppend += "value=\"" + coursesTable[i].id + "\"> [" + "<font color=\"red\">" + coursesTable[i].id + "</font>" + "] : " + coursesTable[i].name;
+                    }
+                    else{
+                    	htmlToAppend += "value=\"" + coursesTable[i].id + "\"> [" + coursesTable[i].id + "] : " + coursesTable[i].name;
+                    }
+                    if (String(coursesTable[i].id) == courseid){
```

Comment:
This 'if' statement looks like a repeat of the previous 'if' statement. Can you put the content of this 'if' statement above? That might shorten the code.